ppb/systems/renderer.py

```diff
@@ -15,6 +15,8 @@
 DEFAULT_RESOLUTION = 800, 600
 
 
+# TODO: Move Image out of the renderer so sprites can type hint
+#  appropriately.
```

Comment:
I've been type-hinting with `Asset` to avoid this, but yeah, we have a circular dependency problem.